builder/oracle/classic/step_add_keys.go

```diff
@@ -50,6 +56,10 @@ func (s *stepAddKeysToAPI) Run(_ context.Context, state multistep.StateBag) mult
 func (s *stepAddKeysToAPI) Cleanup(state multistep.StateBag) {
 	// Delete the keys we created during this run
 	keyName := state.Get("key_name").(string)
+	if keyName == "" {
```

Comment:
this is fine the way it is, but I think it's more idiomatic to not set it to an empty string up above, and here to do

```go
keyName, ok := state.Get("key_name")
if !ok {
    return
}
// ...
deleteInput := compute.DeleteSSHKeyInput{Name: keyName.(string)}
```